integration_tests/integration/helpdesk/contactHelpdelpdesk.spec.ts

```diff
@@ -2,30 +2,56 @@ import RequestLinkPage from '../../pages/link/requestLink'
 import Page from '../../pages/page'
 import ScanBarcodePage from '../../pages/scan/scanBarcode'
 import ContactHelpdeskPage from '../../pages/helpdesk/contactHelpdesk'
+import ContactHelpdeskConfirmationPage from '../../pages/helpdesk/contactHelpdeskConfirmation'
 
 context('Contact Helpdesk', () => {
-  beforeEach(() => {
-    cy.task('reset')
-  })
-
-  it('should render Contact Helpdesk page from Legal Sender Journey', () => {
-    cy.visit('/link/request-link')
-    const requestLinkPage = Page.verifyOnPage(RequestLinkPage)
+  let contactHelpdeskPage: ContactHelpdeskPage
+  const journeyTypes = [
```

Comment:
I've organised these Contact Helpdesk integration tests to operate within a loop, where we loop through 'journey types'
Basically we have 2 journeys (Legal Sender and Mail Room) where fundamentally the tests we want to do are the same, but the setup and how we get there is slightly different. Each journey type has a `setupFunction` that sets up the relevant stubs and gets us to the Contact Helpdesk page ready for the tests to start